requirements-docs.txt

```diff
@@ -1,18 +1,19 @@
 # requirements-docs.txt
-graphviz
-linkify-it-py
-myst-parser
-plone-sphinx-theme
-sphinx-autobuild
-sphinx-copybutton
-sphinx-design
-sphinx-examples
-sphinx-notfound-page
-sphinx-reredirects
-sphinx-sitemap
-sphinxcontrib.httpdomain
-sphinxcontrib.httpexample
-# sphinxcontrib-mermaid  # See https://github.com/mgaitan/sphinxcontrib-mermaid/issues/137
-sphinxcontrib-video
-sphinxext-opengraph
-vale==2.30.0
+-e .
+graphviz==0.20.3
+linkify-it-py==2.0.3
+myst-parser==3.0.1
+Sphinx>=7.3.7,<8.0.0
+sphinx-autobuild==2024.4.16
+sphinx-copybutton==0.5.2
+sphinx-examples==0.0.5
+sphinx-notfound-page==1.0.0
+sphinx-reredirects==0.1.3
+sphinx-sitemap==2.6.0
+sphinx_design==0.5.0
+sphinxcontrib-httpdomain==1.8.1
+sphinxcontrib-httpexample==1.1
+# sphinxcontrib-mermaid==0.9.2  # See https://github.com/mgaitan/sphinxcontrib-mermaid/issues/137
+sphinxcontrib-video==0.2.0
+sphinxext-opengraph==0.9.1
+vale>=3.0.0,<4.0.0
```

Comment:
@stevepiercy I would use specific pins, not ranges, for Sphinx and vale. The point of pinning is to avoid surprises. Upgrading a dependency becomes a task that requires a PR, which means that it is explicitly tested before going into effect.

Reply:
Done.